<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="api" timestamp="2023-03-08T09:22:22Z" hostname="01d9847edc02" tests="1" skipped="0" failures="0" errors="0" time="3.974">
    <testcase time="3.974" classname="api" name="API_greets_the_person"/>
  </testsuite>
  <testsuite name="generate_data" timestamp="2023-03-08T09:22:26Z" hostname="01d9847edc02" tests="4" skipped="0" failures="0" errors="0" time="0.051">
    <testcase time="0.0380000000000003" classname="generate_data" name="generate_data_works_as_expected"/>
    <testcase time="0.00199999999999978" classname="generate_data" name="generate_data_works_as_expected"/>
    <testcase time="0.0100000000000007" classname="generate_data" name="generate_data_works_as_expected"/>
    <testcase time="0.000999999999999446" classname="generate_data" name="generate_data_works_as_expected"/>
  </testsuite>
  <testsuite name="hello" timestamp="2023-03-08T09:22:26Z" hostname="01d9847edc02" tests="1" skipped="0" failures="0" errors="0" time="0.067">
    <testcase time="0.0670000000000002" classname="hello" name="hello_greets_the_entity"/>
  </testsuite>
  <testsuite name="inc_prop" timestamp="2023-03-08T09:22:26Z" hostname="01d9847edc02" tests="2" skipped="0" failures="0" errors="0" time="0.039">
    <testcase time="0.0380000000000003" classname="inc_prop" name="inc_prop_works_as_expected"/>
    <testcase time="0.000999999999999446" classname="inc_prop" name="inc_prop_works_as_expected"/>
  </testsuite>
  <testsuite name="prop_trans_inc_dens" timestamp="2023-03-08T09:22:26Z" hostname="01d9847edc02" tests="1" skipped="0" failures="0" errors="1" time="0.011">
    <testcase time="0.0110000000000001" classname="prop_trans_inc_dens" name="prop_trans_inc_prop_works_as_expected">
      <error type="error" message="Error in ``[.data.frame`(data, type_of_event == 1 &amp; time_to_event &lt;= tau)`: object 'type_of_event' not found ('test-prop_trans_inc_dens.R:10')">Error in ``[.data.frame`(data, type_of_event == 1 &amp; time_to_event &lt;= tau)`: object 'type_of_event' not found
Backtrace:
 1. savvyr:::prop_trans_inc_dens(data = df, tau = 4)
      at test-prop_trans_inc_dens.R:10:2
 4. base::`[.data.frame`(...)</error>
    </testcase>
  </testsuite>
  <testsuite name="shiny" timestamp="2023-03-08T09:22:26Z" hostname="01d9847edc02" tests="1" skipped="0" failures="0" errors="0" time="3.117">
    <testcase time="3.117" classname="shiny" name="The_Shiny_App_returns_a_proper_greeting"/>
  </testsuite>
</testsuites>
